client/src/pages/Login.js

```diff
@@ -0,0 +1,56 @@
+import logoBike from "./logoBike.svg";
+import Form from "react-bootstrap/Form";
+
+
+const Login = () => {
+	return (
+		<form>
+			<div className="Login-page">
+				<div className="flex-container">
+					<div className="img">
+						<div className="container-image">
+							<img src={logoBike} alt="bike logo" />
+						</div>
+
+						<h4>sign in</h4>
+					</div>
+					<div>
+						<>
```

Comment:
```suggestion
```
It looks like these `div` and `<>` elements are doing nothing — can we delete them?